src/main/java/io/swagslash/gametrackerserver/igdbconsumer/model/IGDBGenre.java

```diff
@@ -0,0 +1,27 @@
+package io.swagslash.gametrackerserver.igdbconsumer.model;
+
+public class IGDBGenre {
+    private String name;
+    private String slug;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getSlug() {
+        return slug;
+    }
+
+    public void setSlug(String slug) {
+        this.slug = slug;
+    }
+
+    @Override
+    public String toString() {
+        return "{ Genre: " + this.name + " }";
```

Comment:
[suggestion] Personal preference: `toString` for pure POJOs should include all fields.